libs/application/templates/driving-license/src/fields/Congratulations.tsx

```diff
@@ -15,42 +13,40 @@ interface PropTypes extends FieldBaseProps {
   field: CustomField
 }
 
+interface name {
+  fullName: string
+}
+
 function Congratulations({
   error,
   field,
   application,
 }: PropTypes): JSX.Element {
+  const name = application.externalData.nationalRegistry?.data as name
   return (
     <Box paddingTop={2}>
       <Box marginTop={2}>
         <ContentBlock>
           <AlertMessage
             type="success"
-            title="Til hamingju"
-            message={`
-              fullnaðarskírteinið þitt hefur verið samþykkt og mun vera tilbúið
-              hjá Sýslumanni í Reykjavík eftir 4 daga. Þegar þú sækir nýtt skírteini
-              þarftu að skila inn bráðabirgðaskírteininu þínu.
-            `}
+            title={`Til hamingju ${name.fullName}`}
+            message={`Umsókn þín um fullnaðarskírteini tókst.`}
```

Comment:
noticed that you've added this to messages.ts but perhaps forgetting to use here?